src/components/vmd-upcoming-days-selector.component.ts

```diff
@@ -0,0 +1,80 @@
+import {LitElement, html, customElement, property, css, unsafeCSS} from 'lit-element';
+import upcomingDaysSelectorCss from "./vmd-upcoming-days-selector.component.scss";
+import {RendezVousDuJour} from "../state/State";
+import {CSS_Global} from "../styles/ConstructibleStyleSheets";
+import {repeat} from "lit-html/directives/repeat";
+import {classMap} from "lit-html/directives/class-map";
+import {Strings} from "../utils/Strings";
+import {format, parse} from "date-fns";
+import {fr} from "date-fns/locale";
+
+@customElement('vmd-upcoming-days-selector')
+export class VmdUpcomingDaysSelectorComponent extends LitElement {
+
+    //language=css
+    static styles = [
+        CSS_Global,
+        css`${unsafeCSS(upcomingDaysSelectorCss)}`,
+        css`
+        `
+    ];
+
+    @property() creneauxQuotidiens: RendezVousDuJour[] = [];
+    @property() dateSelectionnee: string|undefined = undefined;
+
+    constructor() {
+        super();
+    }
+
+    render() {
+        return html`
+          <ul class="days list-group list-group-horizontal">
+            ${repeat(this.creneauxQuotidiens, cq => cq.date, cq => {
+                const appointmentCount = cq.total;
+                return html`
+              <li class="list-group-item ${classMap({
+                selected: this.dateSelectionnee === cq.date, 
+                selectable: this.isSelectable(cq),
+                empty: this.dateSelectionnee !== cq.date && appointmentCount === 0
+              })}" @click="${() => this.jourSelectionne(cq)}">
```

Comment:
hello, pour un peu plus d'accessibilité (testé avec TalkBack -  chrome/android)
```suggestion
              })}" @click="${() => this.jourSelectionne(cq)}"
                @keydown="${(e:KeyboardEvent) => e.key === 'Enter' &&  this.jourSelectionne(cq)}"
                role="button" tabindex="0">
```